packages/components/src/components/dropdown/_dropdown.scss

```diff
@@ -66,6 +66,22 @@
     }
   }
 
+  .#{$prefix}--dropdown--xl {
```

Comment:
Where is this class used? Why was it added?

Reply:
Started with targeted search, but designer told us that such dropdown's size variant should not be just for targeted search but also for dropdown in general. You can try this at Storybook.

Reply:
@asudoh dropdown uses the listbox class for size though. This isn't used at all as far as i can tell?

Reply:
@asudoh am I missing something or are these classes never used?

Reply:
Though there is no demonstration, vanilla users should be able to use the class.

Reply:
@asudoh So we added classes that are strictly meant for a specific framework? That seems...not great.

Reply:
Not for a specific framework, but for a legacy markup. Removing the legacy markup of the dropdown didn't fit in `v10` timeframe.